src/aspire/utils/coor_trans.py

```diff
@@ -68,15 +68,18 @@ def grid_1d(n, shifted=False, normalized=True, dtype=np.float32):
     return {"x": x, "r": r}
 
 
-def grid_2d(n, shifted=False, normalized=True, dtype=np.float32):
+def grid_2d(n, shifted=False, normalized=True, indexing="xy", dtype=np.float32):
     """
     Generate two dimensional grid.
 
     :param n: the number of grid points in each dimension.
     :param shifted: shifted by half of grid or not when n is even.
     :param normalized: normalize the grid in the range of (-1, 1) or not.
+    :param indexing: 'xy' (C) or 'ij' (F),
+    see https://numpy.org/doc/stable/reference/generated/numpy.meshgrid.html
     :return: the rectangular and polar coordinates of all grid points.
```

Comment:
We should probably note the default value in the docstring esp. since it's the one less frequently used in the code ("xy")

Reply:
Good call. Changed for both 2d and 3d.